Simplified.xcodeproj/project.pbxproj

```diff
@@ -6162,7 +6157,7 @@
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				MARKETING_VERSION = 3.7.0;
+				MARKETING_VERSION = 3.6.4;
```

Comment:
Should we keep 3.7.0 for CP branch?

Reply:
I only changed the SimplyE and SimplyE-R2dev targets since those are ones we really care about and to avoid doing this fixing over and over when we merge

Reply:
so in other words 3.6.4 is only used for SimplyECardCreator and SimplyE-noDRM targets.